features/OCM_login.feature

```diff
@@ -0,0 +1,83 @@
+Feature: Login into console.redhat.com
+
+  Scenario: Login into console.redhat.com
+    Given console.redhat.com is accessible
+      And user U1 has access granted to login into console.redhat.com
```

Comment:
@tisnik would it make sense to go one step further and start with a new accounts created on access.redhat.com ? One of the problems we had last time was a persmission problem that happened due to unfinished process of creating the new account which wasn't recognized by our tools